1-js/03-code-quality/01-debugging-chrome/article.md

```diff
@@ -1,71 +1,71 @@
 # Depuração de erros no navegador
 
-Antes de escrevermos código mais complexo, vamos falar de debugging (depuração de erros).
+Antes de escrevermos código mais complexo, precisamos falar de debugging (depuração de erros).
```

Comment:
I do think that this change is quite redundant here since the current text already implies this concordance without using the particule explicitly. Can we keep it the way it is?

Reply:
This seems to be the only point where you disagree but I would appreciate it if you could accept the change. Can it be? Thanks.